File: javascript/src/testharness/testharness.js
import { RequestHandler } from '../anni/requests/requests.js';

export class TestHarness {
  constructor(handler = null) {
    this.handler = handler || new RequestHandler();
  }

  async checkElfLogic(person) {
    return await this.handler.checkElfLogic(person);
  }
}

File: python/src/testharness/testharness.py
from anni.requests.requests import RequestHandler
from anni.types.datatypes import Person

class TestHarness:
    def __init__(self, handler: RequestHandler | None = None):
        self.handler = handler or RequestHandler()

    def check_elf_logic(self, people: list[Person]):
        return self.handler.check_elf_logic(people)